design-system/src/main/java/com/msg/sms/design/component/SmsDialog.kt

```diff
@@ -0,0 +1,110 @@
+package com.msg.sms.design.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
+import com.msg.sms.design.component.button.ButtonState
+import com.msg.sms.design.component.button.SmsRoundedButton
+import com.msg.sms.design.theme.SMSTheme
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+fun SmsDialog(
+    widthPercent: Float? = null,
+    widthDp: Dp? = null,
+    heightPercent: Float? = null,
+    heightDp: Dp? = null,
+    title: String,
+    msg: String,
+    outlineButtonOnClick: () -> Unit,
+    normalButtonOnClick: () -> Unit,
+    onDissMissRequest: () -> Unit
```

Comment:
```suggestion
    onDissMissRequest: () -> Unit = {}
```
onDismissRequest는 특수상황에서만 사용될거같은데 기본값 넣어주는거 어떤가요?

Reply:
반영했습니다!